pkg/cfgstruct/bind.go

```diff
@@ -260,6 +260,10 @@ func bindConfig(flags FlagSet, prefix string, val reflect.Value, vars map[string
 				check(err)
 				flags.Float64Var(fieldaddr.(*float64), flagname, val, help)
 			case reflect.TypeOf(string("")):
+				if field.Tag.Get("path") == "true" {
+					// NB: conventionally unix path separators are used in default values
+					def = strings.ReplaceAll(def, "/", string(filepath.Separator))
```

Comment:
There are `filepath.FromSlash` and `filepath.ToSlash`

Reply:
go to know thanks!